### PR TITLE
Praveen merging code with staging

### PR DESCRIPTION
**Pull request by durgavarapraveen**
praveen did some files and now he is merging it with staging branch.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,72 +1,74 @@
...
-      <Orders/>
-    </Route>
-   
-    </div>
-    </Switch>
+      <center>
+        <Grid className='App' style={{justifyContent: 'center', display: 'flex', flexDirection:'column', width: isMobile ? '100%' : '400px'}}>
+          <Grid style={{width: '100%', position: 'sticky' , top: '0', zIndex: '0'}}>
+            <Navbar />
+            <Switch>
```
Comment (venkateshjnv123) [inline src/App.js]: there is already a switch option above, code your routes accordingly

File: src/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        { number: 7, names: "Delivery Pending" },
+        { number: 8, names: "Delivered" },
+        { number: 9, names: "Confirmed Payment" },
+        { number: 10, names: "After Service Payments" },
+        { number: 11, names: "Warranty Clame" },
+    ]
+
+    const handleOpenNewBid = (event) => {
+        setanchorel(event.currentTarget)
+        console.log("Button clicked")
```
Comment (venkateshjnv123) [inline src/Home/Home.js]: comment console.log

File: src/Navbar/Footer.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import BottomNavigationAction from '@mui/material/BottomNavigationAction';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { IoCallOutline, IoChatbubbleEllipsesOutline } from 'react-icons/io5';
+import WhatsAppIcon from '@mui/icons-material/WhatsApp';
+import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined';
+import { ImHome } from 'react-icons/im';
+import { IconContext } from 'react-icons';
+
+export default function Footer() {
+    const Mobile = useMediaQuery('(max-width:400px)');
```
Comment (venkateshjnv123) [inline src/Navbar/Footer.js]: media queries - good use - elaborate it in a code comment

File: src/NewBid/NewBid.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import { IconButton, Radio, RadioGroup } from '@mui/material';
+import CurrencyRupeeOutlinedIcon from '@mui/icons-material/CurrencyRupeeOutlined';
+import Divider from '@mui/material/Divider';
+import InputBase from '@mui/material/InputBase';
+import Button from '@mui/material/Button';
+import styled from '@emotion/styled';
+
+const SubmitButtom = styled(Button) `
```
Comment (venkateshjnv123) [inline src/NewBid/NewBid.js:18]: change submitbuttom name - submitbutton or something else

File: src/components/awaitingConformationPage2.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    const handleCloseCancelBid = (e) => {
+        setCancelBid(false);
+    }
+
+
+    function handleOpenReBid() {
+        setReBid(true);
+        console.log('Clicked rebid')
```
Comment (venkateshjnv123) [inline src/components/awaitingConformationPage2.js:45]: comment console.log in all pages

**Review by venkateshjnv123 — COMMENTED** [review]
please change the following and commit again

File: src/Home/Home.js (hunk 2 above)
Comment (durgavarapraveen) [reply src/Home/Home.js]: I removed Home folder.

**Review by venkateshjnv123 — APPROVED** [review]

**Pull request merged**